brand/models/res_brand.py

```diff
@@ -8,12 +8,14 @@ class ResBrand(models.Model):
     _name = "res.brand"
     _description = "Brand"
 
+    _inherits = {"res.partner": "partner_id"}
```

Comment:
@bizzappdev That's the aim of `delegate` field attribute. No need of this I think